src/WebJobs.Extensions.Http/Routing/WebJobsRouter.cs

```diff
@@ -51,12 +51,6 @@ private void InitializeRouteCollections()
 
         public Task RouteAsync(RouteContext context)
         {
-            // /admin/* routes should not be allowed to be overriden by proxies.
-            if (context.HttpContext.Request.Path.StartsWithSegments(new PathString("/admin"), System.StringComparison.OrdinalIgnoreCase))
```

Comment:
This is really not needed as it brings the knowledge from ScriptHost into routing APIs and we are already doing a check in ScriptHost for this.
The check is here: [https://github.com/Azure/azure-functions-host/blob/dev/src/WebJobs.Script/Host/ScriptHost.cs#L765](https://github.com/Azure/azure-functions-host/blob/dev/src/WebJobs.Script/Host/ScriptHost.cs#L765)